components/apimgt/org.wso2.carbon.apimgt.hostobjects/src/main/java/org/wso2/carbon/apimgt/hostobjects/APIStoreHostObject.java

```diff
@@ -3848,6 +3902,46 @@ public static NativeArray jsFunction_getApplications(Context cx, Scriptable this
         return myn;
     }
 
+    public static NativeArray jsFunction_getLightWeightApplications(Context cx, Scriptable thisObj, Object[] args,
+            Function funObj)
+            throws ScriptException, APIManagementException {
+
```

Comment:
Empty line not needed

Reply:
fixed with d29bcf39da83a26f7260cde8a8aa089a58ee229b